docs/advisories/etcd-manager-certificate-expiration.md

```diff
@@ -50,3 +59,9 @@ We have two suggested workflows to upgrade etcd-manager in your cluster. While b
    kops update cluster --yes
    kops rolling-update cluster --instance-group-roles=Master --force --cloudonly
    ```
+3. If you are doing this after the certificate has expired, then your Kubernetes API is probably down. Updates to the cluster using `kops` and `kubectl` will not work. In this situation, the following is the recommended approach:
+- Update the etcd image to `kopeio/etcd-manager:3.0.20200428` in `etcd-main.manifest` and `etcd-events.manifest` in all of the master nodes. These static manifests are usually present under `/etc/kubernetes/manifests/`.
+- Apply the changes with a kops rolling update:
+  ```
+  kops rolling-update cluster --instance-group-roles=Master --force --cloudonly
+  ```
```

Comment:
I don't understand how this is any different from the previous (2). The part of editing the manifests is just a workaround that should not be part of this advisory. We actually removed it in the past due to mistakes made by users when editing the manifests.

Reply:
Purpose of writing it as a third option is for helping the users who are referring this document to recover from an ongoing kubernetes api downtime due to expired etcd certificates.

We wasted sometime in figuring out how to update etcd certificate using kops when k8s api is down.

I am not sure where to put this exactly, please free to co author the PR and take it live if you feel it can be useful.

Reply:
I am actually curious why
```
kops update cluster --yes
kops rolling-update cluster --instance-group-roles=Master --force --cloudonly
```
didn't work.

Reply:
I remember we were seeing errors since k8s api was down, don't have the terminal open now to confirm what exactly. May be we ran it without cloudonly. Not really sure. If we use cloudonly option will the manifests in the node get updated without talking to k8s api?

Reply:
Also we were using kops1.15.3 to do the update

Reply:
Yes [cloudonly](https://github.com/kubernetes/kops/issues/7507#issuecomment-527502923) was sufficient. Will remove this then.

Reply:
Thanks :)